deploy/helm/stac-scaler/values.yaml

```diff
@@ -0,0 +1,156 @@
+# Default values for stac-scaler.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+parallelism: 1
+jobCleanupTimeSeconds: 600
+activeDeadlineSeconds: 600
+
+namespace: pgstac
+serviceBusConnectionString: "service-bus-conn-string"
```

Comment:
If there's no reasonable default, I would leave this blank. You can use the `required` jinja filter when you use this property to have the deployment fail if the property is not specified. In addition, however, see my comments about whether this could be pulled from a key vault instead.

Reply:
Removed this parameter as it is no longer used.